### ISSUE TITLE
How can we download test data solely?

### ISSUE THREAD

**Issue by JunMa11** [neheller/kits19#32]
Dear @neheller ,

Thanks for releasing the test data. It seems they are merged with training data.
How can we download the interpolated test data **solely**?

Best,
Jun

**Comment by neheller** [comment]
If you have already cloned the repository for the training data on branch `interpolated`, you can simply run `git pull origin interpolated`. If I understand Git LFS correctly, this will download the full test files for the `interpolated` branch and only small references to test files on the `master` branch.

**Comment by JunMa11** [comment]
Dear @neheller ,
Thanks for your quick reply. It works, but each downloaded file only has `1KB`. What command should I use to get the complete image volume?

![image](https://user-images.githubusercontent.com/19947331/61257933-370fb580-a741-11e9-948c-516204e7000a.png)

**Comment by neheller** [comment]
That's odd. I assume you installed Git LFS to get the data in the first place, correct?

**Comment by JunMa11** [comment]
Yes, you are right.

**Comment by JunMa11** [comment]
BTW, would it be possible to use google drive to release the test data? @neheller 
It would be easier to download.

**Comment by neheller** [comment]
I'm running some tests but I can't seem to reproduce the problem. I'm sorry you're having this issue. I'll look into some alternate methods, but it might take me a day or two to figure something out. Once again, my apologies for the inconvenience.

**Comment by JunMa11** [comment]
Hi @neheller , thank you so much for your help.
Also, I really appreciate your great contribution to this excellent challenge.
Best,
Jun

**Comment by luciaL** [comment]
I have the same problem as you.  After git pull origin interpolated, the cases downloaded before became 1KB too which means I need to download again. Have you figure out how to make it right?

**Comment by luciaL** [comment]
I have the same problem as you. After git pull origin interpolated, the folders of the test cases are empty. Do  you figure out how to make it right?

**Comment by JunMa11** [comment]
Hi @neheller , @luciaL 
I re-download the whole dataset, now it works.
best,
Jun

**Comment by luciaL** [comment]
Thank you so much for your reply.

**Comment by gireeshkbogu** [comment]
Somehow I downloaded all 300 files in one folder. How can I separate the train set from the test set? Also, any light on how the top teams generated the validation set (was it random split, 70:30 split?) would be greatly helpful? thanks
